mongo/integration/unified/entity.go

```diff
@@ -123,72 +221,110 @@ func (em *entityMap) addEntity(ctx context.Context, entityType string, entityOpt
 	return nil
 }
 
-func (em *entityMap) gridFSBucket(id string) (*gridfs.Bucket, error) {
+func (em *EntityMap) gridFSBucket(id string) (*gridfs.Bucket, error) {
 	bucket, ok := em.gridfsBuckets[id]
 	if !ok {
 		return nil, newEntityNotFoundError("gridfs bucket", id)
 	}
 	return bucket, nil
 }
 
-func (em *entityMap) bsonValue(id string) (bson.RawValue, error) {
+func (em *EntityMap) bsonValue(id string) (bson.RawValue, error) {
 	val, ok := em.bsonValues[id]
 	if !ok {
 		return emptyRawValue, newEntityNotFoundError("BSON", id)
 	}
 	return val, nil
 }
 
-func (em *entityMap) changeStream(id string) (*mongo.ChangeStream, error) {
+func (em *EntityMap) changeStream(id string) (*mongo.ChangeStream, error) {
 	client, ok := em.changeStreams[id]
 	if !ok {
 		return nil, newEntityNotFoundError("change stream", id)
 	}
 	return client, nil
 }
 
-func (em *entityMap) client(id string) (*clientEntity, error) {
+func (em *EntityMap) client(id string) (*clientEntity, error) {
 	client, ok := em.clientEntities[id]
 	if !ok {
 		return nil, newEntityNotFoundError("client", id)
 	}
 	return client, nil
 }
 
-func (em *entityMap) clients() map[string]*clientEntity {
+func (em *EntityMap) clients() map[string]*clientEntity {
 	return em.clientEntities
 }
 
-func (em *entityMap) collections() map[string]*mongo.Collection {
+func (em *EntityMap) collections() map[string]*mongo.Collection {
 	return em.collEntities
 }
 
-func (em *entityMap) collection(id string) (*mongo.Collection, error) {
+func (em *EntityMap) collection(id string) (*mongo.Collection, error) {
 	coll, ok := em.collEntities[id]
 	if !ok {
 		return nil, newEntityNotFoundError("collection", id)
 	}
 	return coll, nil
 }
 
-func (em *entityMap) database(id string) (*mongo.Database, error) {
+func (em *EntityMap) database(id string) (*mongo.Database, error) {
 	db, ok := em.dbEntites[id]
 	if !ok {
 		return nil, newEntityNotFoundError("database", id)
 	}
 	return db, nil
 }
 
-func (em *entityMap) session(id string) (mongo.Session, error) {
+func (em *EntityMap) session(id string) (mongo.Session, error) {
 	sess, ok := em.sessions[id]
 	if !ok {
 		return nil, newEntityNotFoundError("session", id)
 	}
 	return sess, nil
 }
 
+// GetEventList returns the array of event documents associated with id. This should only be accessed
```

Comment:
The existing functions are not prefixed with "Get". Is there a reason these are?